configuration/configmgr/configmgrlib/SchemaValue.cpp

```diff
@@ -32,6 +32,7 @@ SchemaValue::SchemaValue(const std::string &name, bool isDefined) :
     bitMask.m_deprecated = 0;
     bitMask.m_isUnique = 0;
     bitMask.m_isDefined = isDefined;
+    m_ordinal = 0;
```

Comment:
constructor initializer list?

Reply:
Since this is a native object, direct initialization vs an initializer list is equivalent. There is no construction so direct assignment is OK. If there is a desire to have it in the initializer list, I will move it.